governance/votes/Executive vote - August 22, 2024.md

```diff
@@ -0,0 +1,140 @@
+- [RWA007_A_OUTPUT_CONDUIT](https://etherscan.io/address/0x701c3a384c613157bf473152844f368f2d6ef191)
+- [RWA015_A_INPUT_CONDUIT_JAR_USDC](https://etherscan.io/address/0xb9373c557f3ae8cddd068c1644ed226cfb18a997)
+- [RWA015_A_INPUT_CONDUIT_URN_USDC](https://etherscan.io/address/0xe08cb5e24862ea86328295d5e5c08972203c20d8)
+- [RWA015_A_OUTPUT_CONDUIT](https://etherscan.io/address/0x1e86cb085f249772f7e7443631a87c6bdba2aceb)
+- [RWA009_A_INPUT_CONDUIT_URN_USDC](https://etherscan.io/address/0x08012ec53a7fabf6f33318dfb93c1289886ebbe1)
+
+### Migrate Reserves from PSM-USDC-A to LITE-PSM-USDC-A
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, then the remaining **200 million USDC** in reserve in PSM-USDC-A will be migrated to LITE-PSM-USDC-A.
+
+### Update PSM-USDC-A Fees
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, then PSM-USDC-A fees will be updated as follows:
+
+- PSM-USDC-A [Fee In (`tin`)](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) will be increased by 0.01 percentage points from 0% to **0.01%**.
+- PSM-USDC-A [Fee Out (`tout`)](https://manual.makerdao.com/module-index/module-psm#fee-out-tout) will be increased by 0.01 percentage points from 0% to **0.01%**.
+
+### Update PSM-USDC-A DC-IAM
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, then the PSM-USDC-A DC-IAM parameters below will be updated as follows:
+
+- PSM-USDC-DC-IAM [Maximum Debt Ceiling (`line`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be decreased by 7,500 million DAI from 10,000 million DAI to **2,500 million DAI**.
+- PSM-USDC-A DC-IAM [Target Available Debt (`gap`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-2-target-available-debt-gap-) will be decreased by 180 million DAI from 380 million DAI to **200 million DAI**.
+- PSM-USDC-A DC-IAM [Ceiling Increase Cooldown (`ttl`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-3-ceiling-increase-cooldown-ttl-) will remain unchanged at **12 hours**.
```

Comment:
Why aren't we using billion here and in the sheet? This breaks convention.

Reply:
Discussed on Discord - non-blocking but we do not necessarily agree with this change